be/src/storage/inverted/clucene/clucene_inverted_reader.cpp

```diff
@@ -142,13 +141,13 @@ Status FullTextCLuceneInvertedReader::query_null(OlapReaderStatistics* stats, co
 
         bit_map->swap(*null_bitmap);
 
-        CLOSE_INPUT(dir)
```

Comment:
if dir->openInput failed, the dir will not be closed?

Reply:
I think it will be close. If dir->openInput is failed, a exception will be throwed and inside the catch statement, we use FINALLY_CLOSE_DIR to close the dir